src/main/resources/application.properties

```
@@ -162,7 +162,7 @@ management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
 # Elasticsearch mappings
-elasticsearch.type-mappings.defaultText=Text, TextArea, FixedList, FixedListEdit, MultiSelectList, FixedRadioList, DynamicList, DynamicRadioList, DynamicMultiSelectList,  Region, BaseLocation
+elasticsearch.type-mappings.defaultText=Text, TextArea, FixedList, FixedListEdit, MultiSelectList, FixedRadioList, DynamicList, DynamicRadioList, DynamicMultiSelectList, Region, BaseLocation, PhoneField
```

This should be the CCD field type - which I think (bit please check to confirm) is `PhoneUK`
```suggestion
elasticsearch.type-mappings.defaultText=Text, TextArea, FixedList, FixedListEdit, MultiSelectList, FixedRadioList, DynamicList, DynamicRadioList, DynamicMultiSelectList, Region, BaseLocation, PhoneUK
```